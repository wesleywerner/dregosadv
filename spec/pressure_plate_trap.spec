superbrief
purloin wizard wand, sun key
gonear tomb portal
x column
x hole
look in hole
look
notes on Sowilu
x drawing
turn the key
put key in hole
look in hole
turn the key
again
again
again
again
again
look
gonear waterf portal
gonear tomb portal
take golden egg
undo
purloin stone egg
put stone on altar
take golden egg
