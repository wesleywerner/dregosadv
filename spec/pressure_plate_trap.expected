  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Wizard's Wand: [Purloined.]
  sun key: [Purloined.]
  
> >  Underground Tomb
  
  A stone column stands in the center of the room.
  Some stairs leads upward.
  
> >  The column is round and carved out of stone, it reaches from floor to
  ceiling. My keen eye notices a peculiar hole in the floor, next to the
  column.
  
> >  The hole is no more than a fist deep and a hand wide, it is carved into the
  floor. The hole is in the shape of a sun. I see something in the bottom.
  
> >  I see a glyph engraved in the bottom, I recognize it as Sowilu.
  
> >  Underground Tomb
  
  A stone column stands in the center of the room. There is a sun-shaped hole
  in the floor, next to it.
  Some stairs leads upward.
  
> >  (putting the sun key into the satchel to make room)
  I take my journal from my satchel.
  Sowilu is the rune for the sun, and because of its symbolic power, it is
  rumored that the treasure can only be unlocked with a key of the sun. Here
  on the opposite page is a pencil drawing of the key.
  
  I tear the page out.
  
  I compare the drawing to the hole, it is a perfect fit. It appears that is
  a keyhole.
  
> >  It is a drawing of a key, which resembles a large flat amulet in the shape
  of a sun, the beams are fashioned to serve as the teeth of the key.
  
> >  I turn the key around in my hand, a fine piece of work indeed.
  
> >  (first taking the sun key out of the satchel)
  I put the sun key into the keyhole.
  
> >  In the keyhole is a sun key.
  
> >  The column lowers into the ground with the sound of grinding stone.
  
> >  The column lowers into the ground with the sound of grinding stone.
  
> >  The column lowers into the ground with the sound of grinding stone.
  
> >  The column lowers into the ground with the sound of grinding stone.
  
> >  A low thud echoes through the ground as the column comes to a rest.
  
> >  It does not want to turn any more.
  
> >  Underground Tomb
  
  A stone altar is here in the room, the Golden Egg of Man-Toomba sits
  attractively on top of it!
  Some stairs leads upward.
  
> >  (putting the drawing of the sun key into the satchel to make room)
  I have it! It is magnificent.
  
  The weight lifted from the altar triggers a pressure plate booby trap. Tens
  of tiny poisoned darts shoot out from holes hidden in the runed walls. I
  slump to the floor paralyzed, never to wake up again.
. 
      *** I died ***
. 
  In that game you scored 0 out of a possible 9, in 17 turns.
  
  Would you like to RESTART, RESTORE a saved game, UNDO your last move, give
  the FULL score for that game or QUIT?
> >  Underground Tomb
  [Previous turn undone.]
  
> >  [Purloined.]
  
> >  (the stone egg on the altar)
  I put the stone egg on the altar.
  
> >  (putting the drawing of the sun key into the satchel to make room)
  I have it! It is magnificent.
  
  [I scored my first point! Type SCORE/FULL to list them.]
  
> >